include/NAS2D/Xml/XmlBase.h

```diff
@@ -94,7 +94,14 @@ class XmlBase
 		XML_ERROR_STRING_COUNT
 	};
 
+	struct ParseLocation
+	{
+		int row{0};
+		int column{0};
```

Comment:
In case you were going to submit this to upstream, which I would encourage, this might be using a C+17 feature here. I believe the original author wants to be more general about what standards the code will compile for.

Reply:
TinyXML1 is no longer in development and TinyXML2 explicitly does not use the STL.